packages/app-interbit-io/src/containers/Platform/Platform.js

```diff
@@ -54,7 +54,28 @@ class Platform extends Component {
 
         <Row className="ibweb-mg-xx-lg">
           <Col {...colLayout}>
-            <Quote {...platform.quote} />
+            <div className="ibweb-quote">
```

Comment:
This is pretty much the code same as the <Quote /> component. If you don't want the border on the image, can you modify <Quote /> to take in a className and then override the border settings in _quote.scss? I can give you hand with this.